pol-core/bscript/compiler/astbuilder/FunctionResolver.cpp

```diff
@@ -38,6 +39,12 @@ void FunctionResolver::register_available_user_function(
                                                ctx->EXPORTED() );
 }
 
+void FunctionResolver::register_available_class_decl(
```

Comment:
This whole file will change in the scoping PR, using the approach discussed, keeping track of triplet `(calling_scope, call_scope, name)`